script.js

```diff
@@ -1,66 +1,462 @@
+			else {
+				if (_('check-' + targetElem).checked === true) {
+					_('desc-' + targetElem).classList.add("checkboxChecked");
+				}
+				list.find(elem => elem.id.toString() === targetElem).description = _('desc-' + targetElem).innerText;
+				list.find(elem => elem.id.toString() === targetElem).editMode = false;
+				_('check-' + targetElem).removeAttribute("disabled");
+				_('desc-' + targetElem).contentEditable = false;
+				editimage.classList.remove("saveButtonImage");
+				editimage.classList.add("editButtonImage");
+				editimage.setAttribute("src", './images/edit.png')
+				storeLocal();
+				setTimeout(() => { _("input_notes").focus() }, 200)
+			}
+		}
+		editItem.addEventListener("click", editContent)
+		editimage.addEventListener("click", editContent)
+		description.addEventListener("dblclick", editContent)
+		description.addEventListener("keydown", (e) => {
+			e.stopImmediatePropagation();
+			if (e.code === "Enter") {
+				editContent(e);
+			}
+		})
+
+		//Remove button event
+		let removedItem = _("remove-" + searchResult[eachObj].id);
+		let removeimage = _("removeimage" + searchResult[eachObj].id);
+		function showDeletionModal(e) {
+			e.stopPropagation();
+			_("input_notes").blur();
+			_('removeModal').classList.add("emptyModalParent");
+			let id = e.target.id.replace("removeimage", "").replace("remove-", "");
+			_('confirm').setAttribute("data-id", id)
+		}
+		removedItem.addEventListener("click", showDeletionModal);
+		removeimage.addEventListener("click", showDeletionModal);
+	}
+}
+//search event
+let search = _("searchText");
+function listSearch() {
+	_("printing_list").innerHTML = "";
+	let emptySearch = _("emptySearch");
+	if (emptySearch) {
+		emptySearch.remove();
+		_("list_items").classList.remove("addEmptyImage");
+		_("searchText").classList.remove("noshow");
+	}
+	let searchItem = search.value.trim();
+	if (searchItem)
+		_("x").classList.remove("noshow");
+	else
+		_("x").classList.add("noshow");
+	let searchItemLowercase = searchItem.toLowerCase();
+	let searchResult = list.filter(item => {
+		return item.description.toLowerCase().indexOf(searchItemLowercase) > -1;
+	})
+	searchResultPrint(searchResult);
+}
+_("searchText").addEventListener("keyup", listSearch);
+_("x").addEventListener("click", () => {
+	_("searchText").value = "";
+	listSearch();
+	search.focus();
+	_("x").classList.add("noshow");
+});
+function displayInputBar() {
+	_("floatDiv").classList.add("floatDiv");
+	_("floatButton").classList.add("noshow");
+	_("input_area").classList.add("flexDisplay");
+	_("input_notes").focus();
+}
+function displayFab() {
+	_("overlay").classList.remove("flexDisplay");
+	_("floatDiv").classList.remove("floatDiv");
+	_("floatButton").classList.remove("noshow");
+	_("input_area").classList.remove("flexDisplay");
+}
+_("plus").addEventListener("click", displayInputBar)
+_("floatButton").addEventListener("click", displayInputBar)
+_("floatDiv").addEventListener("click", displayFab);
+function _(id) {
```

Comment:
Let's rename `_` to `$`. Just because [`$` is more popular for this purpose](https://stackoverflow.com/a/1150402/3201403) including in many utility libraries like jQuery.